### DISCUSSION TITLE
Unable to define consistent colors in UnoCss theme

### DISCUSSION THREAD

**Discussion by troikatsy** [unocss/unocss#3654] [Q&A] [answered]
Hi, I am trying to migrate from tailwind to Uno, but facing some unexpected behavior while configuring theme.
Here is my theme config in Uno:
```
theme: {
  colors: {
      'primary': 'blue'
  },
} 
```

Here is my sample HTML:
```
<button class="bg-primary">Blue Button</button>
<button class="bg-primary-anysuffix">Unexpected Blue Button</button>
<button class="bg-primary-any-suffix">Not a Blue Button</button>
```

So if I define blue color to use it with 'bg-primary' class, it should only work with 'bg-primary', but it seems blue color is appled for any class with 'bg-primary' prefix. Ex:  'bg-primary-hover' or  'bg-primary-yourname'

I am not sure if this is a bug or expected behavior.
Please suggest how to configure theme colors in such way that 'bg-primary' works consistently.

**Comment by troikatsy** [comment]
@antfu We are unable to migrate to UnoCSS because of this issue. Any suggestions or workarounds would be highly appreciated. Thank you.

**Reply by Ark21bit** [reply, marked as answer]
try to specify: theme.colors.primary.DEFAULT = 'blue'

**Reply by troikatsy** [reply]
Thank you very much @Ark21bit . You saved our day. By setting 'DEFAULT' we are able to make it work as expected. I think UnoCSS docs should also mention the same.

Anyway, here is working theme config structure for other's reference.

```
theme: {
  colors: {
      'primary':{'DEFAULT': 'blue'}
  }
} 
```
